packages/configs/index.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+	root: true,
+	parser: '@typescript-eslint/parser',
+	env: {
+		node: true,
+	},
+	plugins: ['@typescript-eslint'],
+	extends: [
+		'prettier',
+		'plugin:prettier/recommended',
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'plugin:@typescript-eslint/recommended-requiring-type-checking',
+	],
+	ignorePatterns: [
+		'main.ts',
+		'*.test.*',
+		'__tests__/fixtures/**/*',
+		'setupFile.ts',
+		'__mocks__/**/*',
+		'rollup.config.js',
+		'vite.config.*',
+		'jest.config.*',
+		'lint-staged.config.*',
+		'/dist',
+		'/node_modules'
```

Comment:
Since a lot of these dirs relate to our npm packages, we can probably clean this up a bit for the published version. We can start with the following dirs:
```js
node_modules
build
static
public
```